pypeit/io.py

```diff
@@ -187,7 +188,13 @@ def parse_hdr_key_group(hdr, prefix='F'):
 
     # Convert from dictionary with integer keys to an appropriately
     # sorted list
-    return [values[i] for i in range(max(values.keys())+1)]
+    # JFH try/except added to deal with cases where no header cards were found with the desired
+    # prefix. I'm not sure returning an empty list is the desired behavior, but None will cause
+    # downstream things to crash.
+    try:
+        return [values[i] for i in range(max(values.keys())+1)]
+    except:
+        return []
```

Comment:
Would probably be good to add this as an issue.  Or is it one already that you can reference in the comment?

Reply:
Issued added.